.github/workflows/publish.yml

```
@@ -1,55 +1,24 @@
-      - name: Publish package
-        run: npm publish
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN_PUBLISH_AUTO }}
+jobs:
+  publish-npm:
+    uses: zondax/_workflows/.github/workflows/_publish-npm.yaml@main
+    with:
+      timeout_minutes: 10
+      dry_run: false
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN_PUBLISH_AUTO }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)